docs/suspensive.org/src/pages/docs/react-query/command-line-interface.en.mdx

```diff
@@ -0,0 +1,131 @@
+import { Callout } from '@/components'
+
+# Command-Line Interface (CLI)
+
+The Command-Line Interface (CLI) of @suspensive/react-query is a tool designed to address compatibility issues with @tanstack/react-query. With this tool, you can verify compatibility with @tanstack/react-query and switch to a compatible version of @suspensive/react-query if necessary.
+
+<Callout type="info">The CLI is still an experimental feature and may not function as intended.</Callout>
+
+## When to Use It
+
+Starting from version 2.2.0, @suspensive/react-query supports both versions 4 and 5 of @tanstack/react-query.
+
+Internally, it consists of @suspensive/react-query-4 and @suspensive/react-query-5, which correspond to the respective versions of @tanstack/react-query. (In this document, these version-specific packages are referred to as modules.) The CLI automatically detects the installed version of @tanstack/react-query in your environment and uses the compatible version.
```

Comment:
We need to discuss this part. Since version 2.2.0, @suspensive/react-query has been internally switching between @suspensive/react-query-4 and @suspensive/react-query-5. To make it easier for users to understand, I have referred to this as “modules” in the document. I thought it would be confusing to call them versions 4 or 5 of @suspensive/react-query, as it might be mistaken for the actual package versions.

I think “modules” or “adapters” would be good terms. Do you have any better ideas?

Reply:
@manudeli What do you think about this opinion? I'm concerned that calling `@suspensive/react-query-4` and `@suspensive/react-query-5` "versions" in the documentation might cause confusion with the actual `@suspensive/react-query` package versions. I think it would be better to refer to them as "modules" or "adapters."